src/backend/tests/test_crawler_expunger.py

```diff
@@ -254,3 +254,24 @@ def test_expunger_for_record_with_mj_over_21(record_with_mj_over_21):
             status=EligibilityStatus.ELIGIBLE, reason="Eligible now", date_will_be_eligible=date(2001, 3, 3)
         )
     }
+
+@pytest.fixture
+def record_with_juvenille_and_traffic_charges():
+    return CrawlerFactory.create(
+        JohnDoe.RECORD,
+        {
+            "X0001": CaseDetails.CASE_PARKING_VIOLATION,
+            "CASEJD1": CaseDetails.CASE_MJ_CONVICTION,
+        },
+    )
+
+# FIXME: Unclear to me what the best way to create a minimal test case
+# representing a record that would pass under the old logic but fail under this
+# revised logic
+def test_expunger_for_record_with_non_blocking_other_charges(record_with_mj_over_21):
```

Comment:
Use `YoungDoe.SINGLE_CASE_RECORD` with a convicted CivilOffense charge. See `record_with_mj_under_21`.

Reply:
@wittejm had pointed me towards using the factories that are used in some of the other test suites; is there a preference one way or the other? 

I had started working on their suggestion, but was having a bear of a time getting an `MarijuanaUnder21` charge to be produced (even when injecting a sufficiently young birth year into the Case/CaseSummary) - should it only be tagged as such after passing through the Expunger, since the factories don't seem to actually run through the charge analyzer?

Reply:
Oh you're right, sorry, that may have been a bad suggestion. If all the required HTML fixtures are there, then that makes sense to use those.

Reply:
I don't have a preference. Using the CrawlerFactory will require you to create a new `CaseDetails.CASE_MJ_CONVICTION_WITH_CIVIL_OFFENSE` charge. Or you could just modify the existing `CaseDetails.CASE_MJ_CONVICTION` to include a civil offense charge.

Reply:
The right way to do either of those would be to create a new HTML file, correct?

Reply:
Indeed. Make sure to look at the rendered HTML of existing examples. It should just involve copy/paste, add a new line for the charge, and then a new line for the disposition. The events part is irrelevant in this context.